app/services/project_search_service.rb

```diff
@@ -12,6 +12,10 @@ def search(query)
       return search_by_words(query)
     end
 
+    if establishment_number_pattern(query)
```

Comment:
Now it really feels like a case statement! Perhaps the default is to return no results because we couldn't infer what the users query is?